Jenkinsfile

```diff
@@ -176,7 +188,8 @@ pipeline {
 							'gtk.linux.ppc64le' ,\
 							'gtk.linux.x86_64' ,\
 							'win32.win32.aarch64' ,\
-							'win32.win32.x86_64'
+							'win32.win32.x86_64' ,\
+							'gtk.linux.riscv64'
```

Comment:
A little nitpick:
Lets keep the OS entries group and add `'gtk.linux.riscv64' ,\` after the `gtk.linux.ppc64le` entry.